frontend/views/pages/Contributions.vue

```diff
@@ -104,6 +106,7 @@ page(pageTestName='contributionsPage' pageTestHeaderName='contributionsTitle')
             contribution-item(
               :what='withCurrency(givingMonetary.total)'
               :who='givingMonetary.who'
+              :whoIds='givingMonetary.whoIds'
```

Comment:
Minor: in this file there's a `data-test='revceivingList'` which should be spelled `receivingList`